.github/workflows/codeql.yml

```
@@ -13,16 +13,13 @@ name: "CodeQL Advanced"
 
 on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)